declarations_site/catalog/views.py

```diff
@@ -96,8 +98,6 @@ def search(request):
         search = base_search.query(
             "multi_match",
             query=query,
-            # Not ideal yet, as some of those fields aren't analyzed
-            type="cross_fields",
```

Comment:
But yeah, even this query seems to have twice more results on local (with analyzer) than prod if used in "cross_fields" query. So I'll leave it.